tests/testprogs/uprobe_test.c

```diff
@@ -12,12 +12,13 @@ struct Foo
   int c[3];
 };
 
-int function1(int *n, char c __attribute__((unused)))
+int uprbeFunction1(int *n, char c __attribute__((unused)))
```

Comment:
I changed the names of these functions be more specific as the new tests I added were picking up matching functions from shared libs when testing locally.

Reply:
Why not `uprobeFunction1`?